packages/renderer/package.json

```diff
@@ -9,7 +9,7 @@
   "private": false,
   "scripts": {
     "build": "yarn build:dist && yarn build:lib",
-    "build:dist": "rimraf dist && webpack",
+    "build:dist": "rimraf dist && PACKAGE_NAME=$(echo \"${PWD##*/}\") webpack",
```

Comment:
why `echo`? not something like `cat`
actually, I'm not sure but just interesting why